utils/doctrine-annotation-parser-syncer/src/Rector/ClassMethod/LogIdentifierAndResolverValueInConstantClassMethodRector.php

```diff
@@ -130,8 +127,12 @@ private function createAssignOriginalIdentifierExpression(): Expression
 
     private function createStaticCallExpression(Variable $identifierVariable, Variable $resolvedVariable): Expression
     {
-        $args = [new Arg($identifierVariable), new Arg($resolvedVariable)];
-        $staticCall = new StaticCall(new FullyQualified(ResolvedConstantStaticCollector::class), 'collect', $args);
+        $arguments = [$identifierVariable, $resolvedVariable];
+        $staticCall = $this->nodeFactory->createStaticCall(
+            ResolvedConstantStaticCollector::class,
```

Comment:
This is actually a Rector class, that is always present. It should be used in native way to keep in sync with future renames